challenge.md

```diff
@@ -32,19 +32,20 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
- 
+ Relative.   It has a slash in front of it so there are more directories in front of it.
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
- 
+ Absolute.  The file exists outside of my computer so it needs to know exactly where it is.
```

Comment:
Right answer, wrong reason. All URLs are Absolute paths - they need to be, in order for multiple users to all be able to use the same path (i.e. web address) to refer to the same resource.